functional-tests/fn_bridge_reassignment.py

```diff
@@ -0,0 +1,73 @@
+import time
+
+import flexitest
+
+import testenv
+from constants import UNSPENDABLE_ADDRESS
+from fn_bridge_withdraw_happy import (
+    DEPOSIT_AMOUNT,
+    check_initial_eth_balance,
+    deposit_twice,
+    do_withdrawal,
+    setup_services,
+)
+
+
+@flexitest.register
+class BridgeWithdrawReassignmentTest(testenv.StrataTester):
+    """
+    Makes two DRT deposits, then triggers the withdrawal.
+    The bridge client associated with assigned operator id is stopped.
+    After the dispatch assignment duration is over,
+    Check if new operator is being assigned or not
+    """
+
+    def __init__(self, ctx: flexitest.InitContext):
+        # Example: we spin up 5 operators
+        ctx.set_env(testenv.BasicEnvConfig(101, n_operators=5))
+
```

Comment:
Do we need 5? What if we do it with two and take one out? Or maybe even 3?
The tests should be as minimal as possible with respect to resources to replicate the context/environment that we need to test.

Reply:
We discovered that doing it with just 2 breaks some stuff.  Created a ticket (see comment) for that.